app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -297,6 +297,13 @@ function handleCommentSubmit(event) {
           } else {
             showUserAlertModal('Error posting comment', 'Your comment could not be posted due to an error: ' + errorReponse.error, 'OK')
           }
+        }).catch(function parseError(error){
+            if(responseStatus === 503){
+              showUserAlertModal('Error posting comment', 'your comment could not be posted due to an error: HTTP 503 SERVER ERROR' , 'OK')
+            }else{
+              showUserAlertModal('Error posting comment', 'your comment could not be posted due to some unknown error', 'OK')
+            } 
+            
```

Comment:
It works well! A note:

I don't think we should tell the user, which can't be at all technical, that the error is a 503, but in general I don't think we should actually make a distinction between any of the 500 errors raised by `response.json()`. I don't think we need the `if` condition, we can just display:

```ruby
showUserAlertModal('Error posting comment', 'Your comment could not be posted due to a server error', 'OK')
```

This way we don't alarm the user but we tell them... something.

What do you think?

Reply:
What @rhymes said. 😎

Reply:
Got it! i have made the change.